Samples/Insights/cpp-win32/README.md

```diff
@@ -0,0 +1,14 @@
+# Instructions
+This sample makes use of the Insights headers and library to define data collection events that can be used to track different events in the application.

```

Comment:
>. [](http://example.com/codeflow?start=152&length=1)

I think it would be good to make it clear to the app devs that the Insights library can't be used by them to gather telemetry from the usage of their app in the wild. That is, this data that's collected will not be pushed up to the Microsoft servers and made available to them.

Reply:
I don't want to introduce any divergence from our agreement in terms of the Tracelogging API. I am inclined to leave this as such. @MythiliM : Do you have any input on what must be documented?